src/components/ConditionalFilter/API/initialState/page/InitialPageState.ts

```diff
@@ -0,0 +1,33 @@
+import { ItemOption } from "@dashboard/components/ConditionalFilter/FilterElement/ConditionValue";
+import { UrlToken } from "@dashboard/components/ConditionalFilter/ValueProvider/UrlToken";
+
+export interface InitialPageState {
+  pageTypes: ItemOption[];
+}
+
+export class InitialPageStateResponse implements InitialPageState {
+  constructor(public pageTypes: ItemOption[] = []) {}
+
+  public static empty() {
+    return new InitialPageStateResponse();
+  }
```

Comment:
question: do we even need this static method? it does exactly the same thing as constructor